src/desktop/apps/consign/client/__tests__/analyticsMiddleware.jest.ts

```diff
@@ -47,12 +48,12 @@ describe("analyticsMiddleware", () => {
       payload: { errorType: "convection_create" },
     })
 
-    expect(triggerStub.callCount).toBe(1)
-    const analyticsArgs = triggerStub.firstCall.args
-    expect(analyticsArgs[0]).toBe("consignment:submission:error")
-    expect(analyticsArgs[1]).toEqual({
-      type: "convection_create",
+    expect(window.analytics.track).toHaveBeenCalledTimes(1)
+    expect(
+      window.analytics.track
+    ).toBeCalledWith("consignment:submission:error", {
```

Comment:
@emmadickson @damassi sorry i did not read the comment close enough-- yes the use is inconsistent, we should only use `snake_case` naming for events (the old syntax is for mediator). This event will need its name changed.